scripts/reborn_webui_v2_live_qa/run_live_qa.py

```
@@ -1774,12 +1815,7 @@ async def _resolve_webui_approval_gate(
 
 
 def _slack_bot_token(config_text: str, extra_env: dict[str, str]) -> str | None:
```

Ruff (0.15.20)

[warning] Unused function argument: `config_text`

(ARG001)